kano_settings/parental_config.py

```diff
@@ -153,8 +156,19 @@ def __init__(self, size_x=250, size_y=25):
         EditableList.__init__(self, size_x=size_x, size_y=size_y)
 
     @staticmethod
-    def _sanitise_site(site):
-        return site.replace(' ', '').lstrip('www.')
+    def _sanitise_site(url):
+        parse = urlparse(url)
+
+        # enforcing some standards (e.g. http://)
+        if parse.scheme == '' or parse.netloc == '':
```

Comment:
I see your point and true, the protocol isn't important. However, as per your example, the `netloc` goes into the `path` field. That means that `www.reddit.com/r/subreddit` would be in the path. I don't believe a simple url sanitation needs to turn into a full blown url validator that would require extensive testing with a little effort on their part.